Formula/ignition-math6.rb

```diff
@@ -16,6 +16,7 @@ class IgnitionMath6 < Formula
   depends_on "doxygen" => :build
   depends_on "eigen"
   depends_on "ignition-cmake2"
+  depends_on "pybind11"
```

Comment:
```suggestion
  depends_on "pybind11" => :build
```
Is pybind11 a build dependency only? Looking into the debian packaging seems so to me.

Reply:
the `:build` flag ensures that a dependency is installed when building bottles but not when a bottle is installed. In order to ensure that we have `pybind11` installed in our CI, we need to upgrade our CI scripts. I will accept this change and open a PR to fix the release-tools scripts accordingly